### DISCUSSION TITLE
Support for Markov Chains?

### DISCUSSION THREAD

**Discussion by Ahsoka** [KaTeX/KaTeX#3706] [Q&A] [answered]
Hello,

I was wondering if KaTeX had support for Markov chain diagrams?  If so, could you please give me a simple example or link to some example(s) of how to create them?  If not, is this perhaps planned for a future version?

Thank you,
@Ahsoka

**Comment by ronkok** [comment, marked as answer]
The LaTeX packages for Markov chain diagrams all depend, as far as I know, on the `tikz` package. There is a [long-standing issue](https://github.com/KaTeX/KaTeX/issues/219) requesting support of `tikz`, but that would be a very large effort.

I'm afraid Markov chain diagrams are not coming, at least not any time soon.
